test/scenario/execution_layer_rewards_after_the_merge.test.js

```diff
@@ -32,8 +33,35 @@ const makeAccountingReport = ({ refSlot, numValidators, clBalanceGwei, elRewards
   extraDataHash: ZERO_HASH,
   extraDataItemsCount: 0,
 })
-
-contract.skip('Lido: merge acceptance', (addresses) => {
+function calcReportDataHash(reportItems) {
```

Comment:
can we import these helpers somehow?
It would be less painful to update the func signature in the future if needed

Reply:
done